SDK Showcase/SDK Showcase/Model/SDKConfigModel.swift

```diff
@@ -0,0 +1,22 @@
+//  Copyright © 2025 Onewelcome Mobile Identity. All rights reserved.
+
+struct SDKConfigModel {
+    let dictionary: [String: String]
+    var appIdentifier: String?
```

Comment:
Isn't better to have immutable struct instance and create new one when some changes are needed? I guess that is the main purpose of using struct over class.

Reply:
https://github.com/onewelcome/omi-sdk-showcase-ios/pull/4#discussion_r2036492154

Reply:
these vars were redundant actually.